build.sbt

```diff
@@ -186,7 +96,6 @@ scalacOptions := Seq(
   "-unchecked",
   "-deprecation",
   "-feature",
-  "-Xfatal-warnings",
   "-Xlint:unsound-match",
```

Comment:
I can re-enable it. I always disable it when working on something complicated, because of unused imports. I'm adding and removing code back and forth, and going to imports for each such change kills my concentrations and infuriates me.

Reply:
There is a keystroke in IntelliJ that takes care of unused imports :)